### PR TITLE
Quick-fix for #100

### PR DESCRIPTION
**Pull request by torfjelde**
While I'm finishing up #99, this works as a quick-fix for #100.

### CODE REVIEW

**Comment by mohamed82008** [comment]
As soon as tests pass, let's merge this.

**Comment by torfjelde** [comment]
Test fails:
```julia
MethodError: no method matching logpdf_with_trans(::Product{Discrete,Bernoulli{Tracker.TrackedReal{Float64}},Array{Bernoulli{Tracker.TrackedReal{Float64}},1}}, ::Array{Array{Int64,1},1}, ::Bool)
```
But should this even be tested? Are we supporting `logpdf_with_trans(d::MultivariateDistribution, x::Vector{Vector{T}}, ::Bool)` now?

**Comment by torfjelde** [comment]
So I'm quite confused.. Locally I've commented out the lines
https://github.com/TuringLang/Bijectors.jl/blob/8f53007ffd3b3eb9b43567ff17895102802846dd/test/ad/distributions.jl#L89-L92
but when I do `] test` I _still_ get the error
```julia
Multivariate discrete distributions: Error During Test at /home/tor/.julia/dev/Bijectors/test/ad/distributions.jl:521
  Got exception outside of a @test
  MethodError: no method matching logpdf_with_trans(::Product{Discrete,BetaBinomial{Tracker.TrackedReal{Float64}},Array{BetaBinomial{Tracker.TrackedReal{Float64}},1}}, ::Array{Array{Int64,1},1}, ::Bool)
```
:confused:

**Comment by mohamed82008** [comment]
This is weird, I will take a look.

File: src/compat/distributionsad.jl [hunk 1]
```diff
@@ -259,6 +259,23 @@ function link(
...
+)
+    return mapvcat(dist.v, x) do dist, x
+        invlink(dist, x)
+    end
+end
+function invlink(
+    dist::VectorOfUnivariate,
+    x::AbstractMatrix{<:Real},
+)
+    return mapvcat(eachcol(x)) do x
```
Comment (mohamed82008) [inline src/compat/distributionsad.jl:274]: This should be `maphcat`.

Comment (mohamed82008) [reply src/compat/distributionsad.jl:274]: Same for `link`

**Comment by mohamed82008** [comment]
Fixed. Let's wait for tests to pass then merge.

**Pull request merged**